File: ExamStorm.DataManagement/Models/UserModel.cs
using ExamStorm.DataManager.Models.Exam;
using System;
using System.Collections.Generic;

namespace ExamStorm.DataManager.Models
{
    public class UserModel : BaseModel
    {
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public UserRole Role { get; set; } = UserRole.Common;
        public string Password { get; set; }
        public List<ExamResultModel> ExamResults { get; set; } = new List<ExamResultModel>();
    }

    public enum UserRole
    {
        Common = 1,
        Moderator = 2
    }
}

File: ExamStorm/ClientApp/src/app/models/UserModel.ts
import { BaseModel } from "./BaseModel";

export class UserModel extends BaseModel {
    constructor(id: string, email: string, fName: string, lName: string, role: string) {
        super(id);
        this.email = email;
        this.firstName = fName;
        this.lastName = lName;
        this.role = role;
    }

    email: string;
    firstName: string;
    lastName: string;
    role: string;
}